src/control/server/server.go

```diff
@@ -35,12 +35,6 @@ import (
 	"google.golang.org/grpc"
 )
 
-func main() {
```

Comment:
Is there no longer a need to perform the os.Exit(1) on error?  Or did it move somewhere I may have missed?

Reply:
https://github.com/daos-stack/daos/pull/664/files#diff-bb8c5675ada3c31026ea839e31fdbf64R13

Reply:
Yeah, the idea here is to make an actual server package and move the main function into cmd/daos_server/main.go.

In the short term, this is mostly just cosmetic, but it lays groundwork for making some of this code reusable across multiple binaries and other reorganization possibilities.